arch/arm/boot/dts/bcm2711.dtsi

```diff
@@ -14,6 +14,7 @@
 
 	vc4: gpu {
 		compatible = "brcm,bcm2711-vc5";
+		raspberrypi,firmware = <&firmware>;
```

Comment:
I think this would be better off in bcm2711-rpi.dtsi, since it isn't necessary for all the other non-rpi uses of BCM2711.